Jenkinsfile

```diff
@@ -228,6 +228,7 @@ def winTest(String pythonVersion) {
                 pip freeze
                 python setup.py install
                 coverage run -m unittest discover -s test -vvv
+                if %errorlevel% neq 0 exit /b %errorlevel%
```

Comment:
Have you tested this structure works?

Reply:
I will test it in another PR and remove those changes later in that PR

Reply:
it woks.
```
10:07:31 [windows]   File "C:\Jenkins\ws\mbed-flasher_PR-70
10:07:31 [windows] 
10:07:31 [windows] FAILED (errors=33, skipped=8)
[Pipeline] [windows] echo
10:07:31 [windows] Caught exception: hudson.AbortException: script returned exit code 1
[Pipeline] [windows] step
10:07:32 [windows] [Set GitHub commit status (universal)] FAILURE on repos [GHRepository@30fd12cd[description=mbed device flasher application,homepage=<null>,name=mbed-flasher,license=<null>,fork=false,size=701,milestones={},language=Python,commits={}
```